recipes/mob_suite/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+    name: mob_suite
+    version: 1.4.1
+
+build:
+    number: 1
+    skip: True # [py27 or py34 or py35]
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    string: "py{{CONDA_PY}}"
```

Comment:
Don't need the string here; python versions are already handled in the build strings.

Reply:
OK, will remove it